tunnel.go

```diff
@@ -208,6 +211,7 @@ func (t *Tunnel) TunaPubAddrs() *ts.PubAddrs {
 // function call will overwrite previous accept addresses.
 func (t *Tunnel) SetAcceptAddrs(addrsRe *nkngomobile.StringArray) error {
 	if t.fromNKN {
+		t.config.AcceptAddrs = addrsRe
```

Comment:
Should probably add this after all Listen() calls

Reply:
Yes, modified.